src/pretix/base/models/items.py

```diff
@@ -523,6 +523,12 @@ class Item(LoggedModel):
         verbose_name=_('Allowed membership types'),
         blank=True,
     )
+    require_membership_hidden = models.BooleanField(
+        verbose_name=_('Hide without a valid membership'),
+        help_text=_('Do not show this unless the customer is logged in and has a valid membership. This also means this '
+                    'will never visible in e.g. the widget.'),
```

Comment:
Basically just the „be“ missing in `'will never visible'`, but I changed it a bit more. Regarding the „e.g.“: this suggests there are other places, where the product will not be shown? If not, I removed the „e.g.“

```suggestion
        help_text=_('Do not show this unless the customer is logged in and has a valid membership. Be aware that '
                    'this means it will never be visible in the widget.'),
```